docs/adr/0007-unit-testing-framework.md

```diff
@@ -0,0 +1,75 @@
+# Unit Testing Framework
+
+- Status: Accepted
+  - Supercedees ES-compatible unit test frameworks (ADR#0000)
+- Deciders: Monitor team
+- Date: 2023-07-25
+
+Technical Story: [MNTOR-1823](https://mozilla-hub.atlassian.net/browse/MNTOR-1823)
+
+## Context and Problem Statement
+
+With the transition to Next.js/React, the team wants to consider testing frameworks again, to confirm what works best for the new architecture.
+
+## Decision Drivers <!-- optional -->
+
+- What works best for the current framework, Next.js
+- Which provides the best experience for developers
+
+## Considered Options
+
+- No Change - Ava (ES6+ compatible test runner)
+- Jest (with the Rust Compiler)
+- Jest (with Babel)
+
+## Decision Outcome
+
+Jest (with the Rust Compiler)
+
+This decision was made in an effort to go with what has the most support for our framework (Nextjs).
+
+### Positive Consequences
+
+- Easy to port past tests (Already completed in #3168)
+- Compability with Nextjs framework
+
+### Negative Consequences
+
+- Still doesn't fully support ES6+ modules
+
+---
+
+## Pros and Cons of the Options <!-- optional -->
+
+### No Change - Ava (ES6+ compatible test runner)
+
+- Good, because the system under test and the test code are pure ES modules
+- Good, because the system under test needs no changes
+- Bad, because the unit tests need to be rewritten to use the new non-jest test runner and mocking library
```

Comment:
The (now old) tests were already using them, so would not need rewriting?

Although we would need to figure out how to make it all work with the Next.js toolchain to be able to write tests for the frontend.